libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -685,6 +693,33 @@ public long getLastRefreshTime() {
         public String getInstanceUrl() { return lastNewInstanceUrl; }
     }
 
+    private static void logAnalyticsEvent(String name) {
```

Comment:
All the logAnalyticsEvent look alike - I wonder if we could have a helper method in SalesforceAnalyticsManager that would take in name, context and custom attributes?